### PR TITLE
Feature/jorge/clean

### PR DESCRIPTION
**Pull request by JorgeGreek**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -82,7 +86,7 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
 
     // Retrofit
-    implementation 'com.squareup.okhttp3:logging-interceptor:4.9.0'
+    implementation 'com.squareup.okhttp3:logging-interceptor:5.0.0-alpha.2'
```
Comment (juancoob) [inline app/build.gradle]: @JorgeGreek why do we need the alpha version?

File: app/src/main/java/com/architectcoders/arquitectomarvel/model/database/ResultDao.kt [hunk 2]
```diff
@@ -2,39 +2,28 @@ package com.architectcoders.arquitectomarvel.model.database
...
 @Dao
 interface ResultDao {
 
-    @Insert(onConflict = OnConflictStrategy.REPLACE)
-    suspend fun insertResult(resultRoom: ResultRoom)
-
-    @Insert(onConflict = OnConflictStrategy.REPLACE)
-    suspend fun insertComics(itemComics: ItemComics)
+ //   @Query("SELECT * FROM characterentity ORDER BY name")
+//    suspend fun getCharacters(): List<CharacterEntity>
```
Comment (juancoob) [inline app/src/main/java/com/architectcoders/arquitectomarvel/model/database/ResultDao.kt]: @JorgeGreek could you remove the commented code?

File: data/build.gradle [hunk 3]
```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'java-library'
+    id 'kotlin'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
```
Comment (juancoob) [inline data/build.gradle]: @JorgeGreek could you update the java version?

File: domain/build.gradle [hunk 4]
```diff
@@ -0,0 +1,13 @@
+plugins {
+    id 'java-library'
+    id 'kotlin'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
```
Comment (juancoob) [inline domain/build.gradle]: @JorgeGreek could you update the java version?

File: usecase/build.gradle [hunk 5]
```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'java-library'
+    id 'kotlin'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
```
Comment (juancoob) [inline usecase/build.gradle]: @JorgeGreek could you update the java version?

File: usecase/src/main/java/com/architectcoders/usecase/MyClass.kt [hunk 6]
```diff
@@ -0,0 +1,4 @@
+package com.architectcoders.usecase
+
+class MyClass {
```
Comment (juancoob) [inline usecase/src/main/java/com/architectcoders/usecase/MyClass.kt]: @JorgeGreek could you delete this class?

**Review by juancoob — CHANGES_REQUESTED** [review]
@JorgeGreek thanks! Could you check my comments?

**Review by juancoob — APPROVED** [review]
Cool @JorgeGreek !! Thanks!

**Review by joseantoniocampillo — APPROVED** [review]
That's ok

**Pull request merged**